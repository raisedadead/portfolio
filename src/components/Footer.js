import React from 'react';
import SocialNav from './Nav/SocialNav';
import { Box, Text } from 'mineral-ui';

const Footer = () => (
  <footer>
    <Box as="section" marginHorizontal="auto" width={1 / 2}>
      <SocialNav />
      <Text as="h6" align="center">
        {` `}
        &copy; {new Date().getFullYear()} Mrugesh Mohapatra.
      </Text>
    </Box>
  </footer>
);

export default Footer;
