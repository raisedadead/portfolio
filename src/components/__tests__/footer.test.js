import React from 'react'
import renderer from 'react-test-renderer'
import Footer from '../footer'

describe(`Footer`, () =>
  it(`renders correctly -- according to react-test-renderer`, () => {
    const tree = renderer.create(<Footer />).toJSON()
    expect(tree).toMatchSnapshot()
  })
)
