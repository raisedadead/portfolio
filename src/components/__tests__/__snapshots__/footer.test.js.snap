// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly -- according to react-test-renderer 1`] = `
<footer>
  <section
    className="css-1big75 elghiqq0"
  >
    <nav
      className="css-142ynee e1befqft0"
    >
      <a
        className="e1befqft1 css-sb55b2 e70ypka2"
        data-index={0}
        href="https://twitter.com/raisedadead"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="css-1xssqnp e70ypka1"
        >
          <span
            className="css-pw07op e70ypka0"
          >
            <span
              className="css-1me6504 e1i5p1da0"
            >
              <span
                className="css-1h0pl34 ew65k9p0"
                id="tooltip-1"
              >
                <span
                  className="css-pusqig ew65k9p3"
                >
                  <i
                    className="fa"
                  />
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
      <a
        className="e1befqft1 css-sb55b2 e70ypka2"
        data-index={1}
        href="https://github.com/raisedadead"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="css-1xssqnp e70ypka1"
        >
          <span
            className="css-pw07op e70ypka0"
          >
            <span
              className="css-1me6504 e1i5p1da0"
            >
              <span
                className="css-1h0pl34 ew65k9p0"
                id="tooltip-2"
              >
                <span
                  className="css-pusqig ew65k9p3"
                >
                  <i
                    className="fa"
                  />
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
      <a
        className="e1befqft1 css-sb55b2 e70ypka2"
        data-index={2}
        href="https://blog.raisedadead.com"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="css-1xssqnp e70ypka1"
        >
          <span
            className="css-pw07op e70ypka0"
          >
            <span
              className="css-1me6504 e1i5p1da0"
            >
              <span
                className="css-1h0pl34 ew65k9p0"
                id="tooltip-3"
              >
                <span
                  className="css-pusqig ew65k9p3"
                >
                  <i
                    className="fa"
                  />
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
      <a
        className="e1befqft1 css-sb55b2 e70ypka2"
        data-index={3}
        href="https://freeCodeCamp.org/raisedadead"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="css-1xssqnp e70ypka1"
        >
          <span
            className="css-pw07op e70ypka0"
          >
            <span
              className="css-1me6504 e1i5p1da0"
            >
              <span
                className="css-1h0pl34 ew65k9p0"
                id="tooltip-4"
              >
                <span
                  className="css-pusqig ew65k9p3"
                >
                  <i
                    className="fa"
                  />
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
      <a
        className="e1befqft1 css-sb55b2 e70ypka2"
        data-index={4}
        href="https://linkedin.com/in/mrugeshm"
        onClick={[Function]}
        rel="noopener noreferrer"
        target="_blank"
      >
        <span
          className="css-1xssqnp e70ypka1"
        >
          <span
            className="css-pw07op e70ypka0"
          >
            <span
              className="css-1me6504 e1i5p1da0"
            >
              <span
                className="css-1h0pl34 ew65k9p0"
                id="tooltip-5"
              >
                <span
                  className="css-pusqig ew65k9p3"
                >
                  <i
                    className="fa"
                  />
                </span>
              </span>
            </span>
          </span>
        </span>
      </a>
    </nav>
    <h6
      className="css-pdf1eb e19le4zx0"
    >
       
      Â© 
      2019
       Mrugesh Mohapatra.
    </h6>
  </section>
</footer>
`;
